demos/redpanda/ockam-consumer

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+
+CONSUMER_TOKEN=$(ockam project ticket --attribute role=member)
+
+NAME="consumer"
+ockam identity create "$NAME"_identity
+ockam project enroll $CONSUMER_TOKEN --identity "$NAME"_identity
+ockam node create "$NAME"_node --identity "$NAME"_identity
+ockam kafka-consumer create \
+  --at "$NAME"_node \
+  --project-route /project/default/service/forward_to_broker/secure/api
```

Comment:
if `localhost:4000` is used below, this line should declare `--bootstrap-server 127.0.0.1:4000`

Reply:
the `--project-route` value has to be a route to the outlet node's secure channel listener:

for example:
```
/ip4/127.0.0.1/tcp/6000/secure/api
```